FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -512,14 +591,18 @@ - (void)setAPNSToken:(NSData *)apnsToken type:(FIRMessagingAPNSTokenType)type {
 
   // Notify InstanceID that APNS Token has been set.
   NSDictionary *userInfo = @{kFIRMessagingAPNSTokenType : @(type)};
+  // TODO(chliang) This is sent to InstanceID in case users are still using the deprecated SDK.
+  // Should be safe to remove once InstanceID is removed.
```

Comment:
Couldn't users be using InstanceID long after we remove it?

Reply:
No if we remove it completely from firebase-ios-sdk. if they use previous version of IID it's probably fine as messaging still kept those communications.